books/free-programming-books-langs.md

```diff
@@ -26,6 +26,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Bash](#bash)
 * [Basic](#basic)
 * [BETA](#beta)
+* [BeanShell](#beanshell)
```

Comment:
Try to ensure linter alphabetization. Move the new section to right place too

Reply:
Yeah, sure.Thanks!